arelle/plugin/validate/ESEF_2022/__init__.py

```diff
@@ -0,0 +1,1225 @@
+'''
+Created on June 6, 2018
+Version 2022 created on September 19, 2022
+
+Filer Guidelines:
+  RTS: https://eur-lex.europa.eu/legal-content/EN/TXT/?qid=1563538104990&uri=CELEX:32019R0815
+  ESEF Filer Manual https://www.esma.europa.eu/sites/default/files/library/esma32-60-254_esef_reporting_manual.pdf
```

Comment:
Just a comment, no work needed. Its really unfortunate that ESMA overwrote the older url with the new data. It would have been nice to be able to keep the documentation on the specs separate.

On a side note does anyone know if a new RTS has been issued to accompany the new guidelines?

Reply:
Hello,

Here is the consolidated version that puts together all rules on that matter:

https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A02019R0815-20220101

Maybe we can put that version in the file comments...

Greg